catalog/resourcemanagement/resourcemanagement-usage-ui/src/main/webapp/js/HandlebarsHelpers.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define*/
+define([
+        'icanhaz',
+         'jquery'
+    ],
+    function (ich, $) {
+        "use strict";
+        var helper, helpers = {
```

Comment:
Since there's only one helper, might be better to be more explicit.
```javascript
ich.addHelper('select', function (value, options) {
                var $el = $('<select />').html( options.fn(this) );
                $el.find('[value="' + value + '"]').attr({'selected':'selected'});
                return $el.html();
});
```
If you're sure there will be more helpers, I would prefer the following pattern so that it's more readable.  

```javascript
[{
     name: 'select',
     callback: function (value, options) {
          var $el = $('<select />').html( options.fn(this) );
          $el.find('[value="' + value + '"]').attr({'selected':'selected'});
          return $el.html();
}].forEach(function(helper){
     ich.addHelper(helper.name, helper.callback);
});
```